plugins/lookup/hashi_vault.py

```diff
@@ -235,6 +246,16 @@
 - name: authenticate with aws_iam_login
   ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret/hello:value', auth_method='aws_iam_login' role_id='myroleid', profile=my_boto_profile) }}"
+
+
+# Defining action to be performed
+# Uses the hvac Client for HashiCorp's Vault to extend the integration with HashiCorp's Vault
+- name: Read a secret (default)
+  ansible.builtin.debug:
+    msg: "{{ lookup('community.general.hashi_vault', 'secret/data/hello action_type=READ token=my_token_var url=http://myvault_url:8200') }}"
```

Comment:
```suggestion
    msg: "{{ lookup('community.general.hashi_vault', 'secret/data/hello action_type=read token=my_token_var url=http://myvault_url:8200') }}"
```
Similar for the second example.

Reply:
of course, well spotted :man_facepalming: